settings.gradle

```
@@ -5,6 +5,45 @@ pluginManagement {
     }
 }
 
+dependencyResolutionManagement {
+    def slangDependenciesVersion = '1.16.0.6211'
+    def analyzerCommonsVersion = '2.13.0.3004'
+    def pluginApiVersion = '10.10.0.2391'
+    def sonarqubeVersion = '10.0.0.68432'
```

Wouldn't it make sense to target [10.6.0.92116](https://github.com/SonarSource/sonarqube/releases/tag/10.6.0.92116)?
```suggestion
    def sonarqubeVersion = '10.6.0.92116'
```